store/postgres/tests/store.rs

```diff
@@ -43,12 +44,22 @@ where
     };
 
     let (sender, receiver) = oneshot::channel();
-    tokio::run(future::lazy(|| {
-        insert_test_data();
-        let result = panic::catch_unwind(|| test());
-        remove_test_data();
-        sender.send(result).map_err(|_| ())
-    }));
+
+    let mut runtime = tokio::runtime::Runtime::new().unwrap();
+    tokio_executor::with_default(
+        &mut runtime.executor(),
+        &mut tokio_executor::enter().unwrap(),
+        |_| {
```

Comment:
I think this whole `tokio_executor::with_default` stuff is actually not necessary, and if you just remove it and leave only the inner `runtime.block_on` it will still work.